### ISSUE TITLE
Posibility of usage ClaymoreDual with nVidia cards

### ISSUE THREAD

**Issue by lukasmagi** [nicehash/NiceHashMiner-Archived#967]
Hello I got problem with my rig (Win10 6x nv1060) when nicehash mine want to mine ETH - Ethereum miner - GenoilFork . I have this problem https://forum.ethereum.org/discussion/10219/windows-10-ethminer-slow-type-writer  + many time not start and freeze or crash (if i runn it maualy -got same results)

But ClamoreDual - Work wery well

Can you give it as option for Nvidia Cards ?

Thx and regards Lukas

**Comment by nvidia3rdparty** [comment]
Yes. Please! Just allow it run. It runs fine when launched manually from the folder. It downloads it automatically but won't use it for some strange reason :/

**Comment by DillonN** [comment]
If you're able to download the source code and compile yourself, NVIDIA support for ClaymoreDual is a one line fix

Simply add `new Algorithm(MinerBaseType.ClaymoreAMD, AlgorithmType.DaggerHashimoto, ""),` as a line under the NVIDIA section of `Devices\GroupAlgorithm.cs`. E.g. I have it on line 213 and it is working fine.

If you are not comfortable compiling yourself then it's probable we will see official support for it soon with an upcoming release

**Comment by lukasmagi** [comment]
Ca you please share that edited file ? ... Iam not good to compile anything

**Comment by yetanotherjson** [comment]
I don't see a file called GroupAlgorithm.cs anywhere in the NiceHashMiner folder, please clarify where to find this.  Thanks!

**Comment by DillonN** [comment]
It is [this](https://github.com/nicehash/NiceHashMiner/blob/master/NiceHashMiner/Devices/GroupAlgorithms.cs) here

**Comment by nvidia3rdparty** [comment]
I tried to compile it with visual studio but it just hangs. I quess I have to wait for official release then.

**Comment by billmcilhargey** [comment]
waiting for an official release too +1

**Comment by 008** [comment]
@0crazyjay0 can you give me compiled version ? want to make some tests

**Comment by billmcilhargey** [comment]
Just to link this issue to the PR that will allow this to be closed once merged https://github.com/nicehash/NiceHashMiner/pull/985

**Comment by gacopl** [comment]
All you need is here ready for you to use
https://github.com/gacopl/NiceHashMiner/releases/tag/1.7.5.13-gaco-mod

**Comment by Sephiroth4269** [comment]
Since people know some more in depth things on this thread, how in blazes do you adjust your parameters? Which JSON file is it and in which folder? I know precisely WHAT I want to change and how, but everytime I try in the files I have found, it doesnt work...
